base_requirements.txt

```diff
@@ -18,6 +18,14 @@ django-filter
 # https://github.com/django-mptt/django-mptt
 django-mptt
 
+# Prometheus metrics client implementation for Django
+# https://github.com/korfuri/django-prometheus
+django-prometheus
```

Comment:
Let's break out the prometheus stuff from the caching work. It really needs to be its own FR with discussion separate from #2647. I'd also much prefer to evaluate caching on its own prior to the introduction of new middleware, which could complicate things.